Soccar [Game-Server]/src/nl/soccar/gameserver/message/handler/PlayerLeaveSessionMessageHandler.java

```diff
@@ -0,0 +1,53 @@
+package nl.soccar.gameserver.message.handler;
+
+import io.netty.buffer.ByteBuf;
+import java.util.logging.Level;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.logging.Level'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)